README.md

```diff
@@ -50,8 +50,15 @@ script:
   - cask exec ert-runner
 ```
 
-This setup builds and tests your Emacs Lisp project on Emacs 24.3, 24.5, 25.1
-and the current Emacs snapshot from Git.
+This setup builds and tests your Emacs Lisp project on Emacs 24.3, 24.5, 25.1,
+25.2 and the current Emacs snapshot from Git.
+
+Note: To use Texinfo you'll need `libperl.so`. For example in Ubuntu you can
+install it this by using command as follows:
+
+```bash
+sudo apt-get install libperl-dev
```

Comment:
Thanks for catching this.  I think you can also add this dependency to our own `travis.yml` in `before_install`, so that we can test it directly.